generator/generator_tests_support/test_feature.hpp

```diff
@@ -150,6 +150,19 @@ class TestPark : public TestFeature
   vector<m2::PointD> m_boundary;
 };
 
+class TestRoad : public TestFeature
```

Comment:
No.
1. All other TestFeatures are not marked as final.
2. In general, we have not been marking classes as final in generator/ and I don't see a reason to start.